public/lib/filters-service.js

```diff
@@ -49,7 +49,7 @@ angular.module('wfFiltersService', ['wfDateService', 'wfTrustedHtml'])
                     $rootScope.$broadcast('getContent');
                 });
 
-                $rootScope.$on('filtersChanged.deadline', function (event, data) { 
+                $rootScope.$on('filtersChanged.deadline', function (event, data) {
```

Comment:
Are there any expected changes here, or is this a product of linting?

Reply:
Just an accidental extra space from linting, no expected changes!